fixtures/external-types/lib/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+use crate_one::CrateOneType;
+use crate_two::CrateTwoType;
+
+pub struct CombinedType {
+    pub cot: CrateOneType,
+    pub ctt: CrateTwoType,
+}
```

Comment:
mhammond mentioned that these fields don't have the greatest names and that he has a similar fixture which can be merged with this one.  My proposal is that we merge this code to main, then one of us can create a PR that merges/improves the fixture.